test/unit/specs/components/SelectedLicense.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
- Please test that the link to the full license text is valid. 
 - I would also like to see tests for licenses other than just BY

Reply:
Please add test for the computed `iconsList` property to be sure that the correct array is being returned based on the shortlicensename. Remember to test this for multiple licenses.

Reply:
Please add test for `watch` to be sure that the link to the full license text updates properly when the shortlicensename property updates.

Reply:
Bonus points if you can find a way to check that the URL isn't dead :)